plugins/validator.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const parser = require('screwdriver-config-parser');
+const schema = require('screwdriver-data-schema');
+const validatorSchema = schema.api.validator;
+const boom = require('boom');
+
+/**
+ * Hapi Validator Plugin
+ *  - Validates screwdriver.yaml and returns the expected execution steps
+ * @method register
+ * @param  {Hapi.Server}    server
+ * @param  {Object}         options
+ * @param  {Function} next
+ */
+exports.register = (server, options, next) => {
+    server.route({
+        method: 'PUT',
```

Comment:
I'm torn.  @jer and I went back and forth.  The reason we went with a PUT is that it's cacheable.  Although a POST makes more sense from the "Providing a block of data, such as the result of submitting a form, to a data-handling process;" perspective.

I'm not picky on this.